### PR TITLE
Remove extra AppVeyor environment check

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary

Removes a check for AppVeyor env
  

## PR Context  
AppVeyor removed (#8686  extension)

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [X] Issue filed:  #8686 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by RDIL** [comment]
codacity?

**Review by bergmeister — CHANGES_REQUESTED** [review]
Sorry, I realised only now that appveyor.yml is now completely gone. This change is ok

**Review by bergmeister — APPROVED** [review]

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
Do we still use appveyor.psm1 ?

**Comment by RDIL** [comment]
Yeah, the Windows VSTS agent imports it.

**Comment by RDIL** [comment]
@iSazonov https://github.com/PowerShell/PowerShell/blob/master/.vsts-ci/windows.yml#L70

**Comment by iSazonov** [comment]
Thanks. I see. Maybe need rename...

**Comment by RDIL** [comment]
I can do that if you would like.

**Comment by iSazonov** [comment]
Not in the PR and if @TravisEz13 agrees.

**Comment by RDIL** [comment]
Alright, if Mr. Plunk gives his 👍 then I'll open a different PR for it.

File: test/packaging/windows/msi.tests.ps1 [hunk 1]
```diff
@@ -74,13 +74,9 @@ Describe -Name "Windows MSI" -Fixture {
     }
     AfterEach {
         if ($Error.Count -ne 0 -and !$uploadedLog) {
-            if ($env:APPVEYOR) {
-                Push-AppveyorArtifact $msiLog
```
Comment (TravisEz13) [inline test/packaging/windows/msi.tests.ps1]: I added a test helper for this, with the conditionals in the helper.

Comment (RDIL) [reply test/packaging/windows/msi.tests.ps1]: Looks like it will work

**Comment by iSazonov** [comment]
@TravisEz13 Could you please merge?

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**